### PR TITLE
Cleanup of CheckStyle suppressions for HazelcastInstanceFactory.

### PR DESCRIPTION
**Pull request by Donnerbart**
Not much to say, there were just some "needs at least one statemen" and a magic number. Very little cleanup needed for a full CheckStyle suppression.

### CODE REVIEW

**Comment by mesutcelik** [comment]
unrelated test failure `MapLoaderFailoverTest.testLoadsAll_whenInitialLoaderNodeRemovedAfterLoading`
verify

**Comment by mesutcelik** [comment]
verify

**Comment by pveentjer** [comment]
verify

**Comment by pveentjer** [comment]
Jenkins messed up again..

File: hazelcast/src/main/java/com/hazelcast/instance/HazelcastInstanceFactory.java [hunk 1]
```diff
@@ -161,20 +160,21 @@ private static HazelcastInstanceProxy constructHazelcastInstance(Config config,
...
-            final int initialWaitSeconds = node.groupProperties.INITIAL_WAIT_SECONDS.getInteger();
+            Node node = hazelcastInstance.node;
+            boolean firstMember = isFirstMember(node);
+            int initialWaitSeconds = node.groupProperties.INITIAL_WAIT_SECONDS.getInteger();
             if (initialWaitSeconds > 0) {
                 hazelcastInstance.logger.info("Waiting "
                         + initialWaitSeconds + " seconds before completing HazelcastInstance startup...");
                 try {
-                    Thread.sleep(TimeUnit.SECONDS.toMillis(initialWaitSeconds));
+                    TimeUnit.SECONDS.sleep(initialWaitSeconds);
```
Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/instance/HazelcastInstanceFactory.java]: If you do a static import of SECONDS it gets even shorter.

**Comment by pveentjer** [comment]
:+1:

**Comment by Donnerbart** [comment]
run-parallel

**Pull request merged**